### ISSUE TITLE
Custom favicon appears during webpage load, and is replaced by default once loading is finished

### ISSUE THREAD

**Issue by Porkepix** [NodeBB/NodeBB#6742]
- **NodeBB version:** 1.7.5
- **NodeBB git hash:** 7df191412de9dfd88c3c9b919b6989fd18b5d445

- **Database type:** mongo
- **Database version:** v2.6.12
- **Exact steps to cause this issue:**
Just load the homepage of a NodeBB with a set favicon
- **What you expected:**
Favicon to hold
- **What happened instead:**
During the loading time, favicon appears.
Then once it finished loading the page, it gets back to default NodeBB favicon.

**Comment by barisusakli** [comment]
Does this happen at https://community.nodebb.org/? Might be already fixed on the latest releases. 1.7.5 is pretty old.

**Comment by nodebb-misty** [comment]
Issue closed due to inactivity.

----

*This is an automated message. If you feel this action was in error, please comment on this issue so it can be looked at again*

**Comment by Porkepix** [comment]
So, our admin updated to latest version this week, and issue is still here.

**Comment by julianlam** [comment]
@Porkepix Does the issue also present in an incognito browser?

**Comment by Porkepix** [comment]
Yes, reproduced right now in a private browsing window on two different computers and in the Tor Browser.

**Comment by barisusakli** [comment]
I can't repro at https://community.nodebb.org/

**Comment by julianlam** [comment]
... is the site reachable publicly? We can check the source html returned.

**Comment by Porkepix** [comment]
I see the issue on https://forum.exodus-privacy.eu.org
I am not admin of the platform, but can talk about it with him.

**Comment by barisusakli** [comment]
https://forum.exodus-privacy.eu.org/assets/favicon.ico?v=jvj1rle6580

The favicon is the default nodebb one

**Comment by Porkepix** [comment]
Yes, but when you laod the page there's one with the association's logo first during load, and then it's replaced by default one.

**Comment by julianlam** [comment]
I think what @barisusakli is trying to say is that there is nothing in the source html that suggests that the association's logo is a favicon. So they need to set the favicon in their site settings. Right now the favicon linked above is just the default NodeBB logo.

**Comment by Porkepix** [comment]
So, our administrator confirmed to me that he did set a favicon and that it was already set at the time I reported this issue.

**Comment by nodebb-misty** [comment]
Issue closed due to inactivity.

----

*This is an automated message. If you feel this action was in error, please comment on this issue so it can be looked at again*

**Comment by julianlam** [comment]
We are unable to confirm this issue as the page source itself does not suggest that a favicon is set.

**Comment by Porkepix** [comment]
@julianlam Well, what else can I do to help for the issue then? The NodeBB admin confirmed he did set such favicon, and you briefly see it during load. I have no idea what else can I do to help on the issue…

**Comment by julianlam** [comment]
The problem is neither @barisusakli nor I see a favicon flash during load so it could be coming from your browser's cache.

To our knowledge the favicon logic is working perfectly fine, if we can't reproduce it locally (or even on your linked site), we can't fix it. Simple as that.

**Comment by Porkepix** [comment]
So, I sadly didn't recorded the loading nor the devtools network waterfall, but it was having 4 pictures lines, one 302 and three 200. One of them was the favicon, with something like touch icon in the name. One was the picture in website header, and one was NodeBB's default favicon.
All reloads were done using ctrl-shift-r, which means it's supposed to ignore any cache. However I could reproduce it at wish on two different computers.

I asked our admin to set again - it was already done - the favicon.

It's now fixed, and display directly and permanently the newly set favicon.
I am however unable to explain nor understand what happened for this problem nor where was it located: two different computers, not supposed to use cache that were affected, now none…

Could somehow the favicon have been unset with an update?
